source/manual/covid-19-services.html.md

```diff
@@ -91,8 +99,9 @@ This is also stored for 2 hours.
 
 ### Data storage
 
-For both applications, when the user submits the form on the "Check
-your answers" page, the application writes user data to the database.
+For all but the find support form, when the user submits the form on
+the "Check your answers" page, the application writes user data to the
+database.
```

Comment:
Might be worth adding when the find support page does this, ie. on submission of the final question